### PR TITLE
[Doc] Update version from 0.4.0 to 0.5.0 on remaining kuberay docs files

### PR DESCRIPTION
**Pull request by architkulkarni**
Some files (ray service doc, kuberay operator doc, kind doc) still were using 0.4.0 in their examples. This PR updates them to 0.5.0.
## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by kevin85421** [comment]
Maybe you can use 0.4.0, v0.4.0, and 0.4 in your VSCode search to make sure all versions are replaced?

**Comment by architkulkarni** [comment]
> Maybe you can use 0.4.0, v0.4.0, and 0.4 in your VSCode search to make sure all versions are replaced?

Yup that's what I did, the rest of them did not make sense to change (e.g. it was referring specifically to 0.4.0 not 0.5.0, or it was using 0.4.0 as an example in some release process doc)

**Comment by kevin85421** [comment]
There seem to be some documents that need to be updated. For example,

* https://github.com/ray-project/kuberay/blob/master/clients/python-client/python_client/__init__.py
* https://github.com/ray-project/kuberay/tree/master/helm-chart/kuberay-apiserver

Do I miss anything?

**Comment by architkulkarni** [comment]
Ah I wasn't sure if I was supposed to update those.  I've updated them now, that should be everything

File: helm-chart/kuberay-apiserver/Chart.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
```
Comment (architkulkarni) [inline helm-chart/kuberay-apiserver/Chart.yaml:18]: @kevin85421 I'm not sure about this one.  Can you confirm if this is supposed to be updated?

**Comment by architkulkarni** [comment]
Updated the rest of the files.

File: helm-chart/kuberay-apiserver/Chart.yaml (hunk 1 above)
Comment (kevin85421) [reply helm-chart/kuberay-apiserver/Chart.yaml:18]: Yes

**Review by kevin85421 — APPROVED** [review]
LGTM. The updates for APIServer and `proto` leave to @scarlet25151 and @Jeffwan.

**Review by Jeffwan — APPROVED** [review]
The changes looks good to me. /lgtm Thanks!

File: proto/kuberay_api.swagger.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "KubeRay API",
-    "version": "0.4.0",
+    "version": "0.5.0",
```
Comment (Jeffwan) [inline proto/kuberay_api.swagger.json:5]: it's generated correctly.

**Pull request merged**